### PR TITLE
Feature/ntgl 271 course delivery mode icons

### PR DESCRIPTION
**Pull request by emilytangg**
- Removed the in-person course filter tabs
- Made the delivery mode icons correspond to course data

### CODE REVIEW

File: client/src/components/CourseSelect.tsx [hunk 1]
```diff
@@ -4,6 +4,8 @@ import React, { useContext, useEffect, useRef, useState } from 'react';
 import { Box, Chip, InputAdornment, TextField, Theme } from '@material-ui/core';
 import { AddRounded, CheckRounded, CloseRounded, SearchRounded } from '@material-ui/icons';
 import { Autocomplete } from '@material-ui/lab';
+import VideocamOutlinedIcon from '@material-ui/icons/VideocamOutlined';
```
Comment (munjoonteo) [inline client/src/components/CourseSelect.tsx]: These two imports can be included in the import from `@material-ui/icons` on line 5

File: client/src/components/CourseSelect.tsx [hunk 2]
```diff
@@ -230,7 +242,7 @@ const CourseSelect: React.FC<CourseSelectProps> = ({ assignedColors, handleSelec
       }
     }
   };
-
+
```
Comment (munjoonteo) [inline client/src/components/CourseSelect.tsx]: Random spaces?

File: server/src/controllers/getCourse.ts [hunk 3]
```diff
@@ -22,120 +22,9 @@ export const getCourse = async (req: Request, res: Response) => {
...
-            },
-          ],
-          notes: [
-            'In 21T1 this course will have online lectures and tutorials. Students who are overseas, or unable to attend campus will be able to complete this course, including all classes and assessments, in the online mode.',
-          ],
-        }
-      : await Database.dbRead(args)
+  const course = await Database.dbRead(args)
+
+  console.log(course)
```
Comment (munjoonteo) [inline server/src/controllers/getCourse.ts]: No `console.log()`s please

**Review by munjoonteo — CHANGES_REQUESTED** [review]

File: client/src/components/CourseSelect.tsx (hunk 2 above)
Comment (emilytangg) [reply client/src/components/CourseSelect.tsx]: Ah I'm not sure why it's like that, will be more careful in future.

Comment (munjoonteo) [reply client/src/components/CourseSelect.tsx]: Prettier should get rid of it for you - have you set `Auto Save` to be `afterDelay` and `Format on Save` to be `true` and `Default Formatter` to be `Prettier` in your VSCode?

Comment (emilytangg) [reply client/src/components/CourseSelect.tsx]: Ahh I forgot to do those things after I installed the Prettier extension, thanks MJ!

**Review by munjoonteo — APPROVED** [review]

**Review by oliverxu42 — APPROVED** [review]

**Pull request merged**